### ISSUE TITLE
Change the warning message when gov limit reached

### ISSUE THREAD

**Issue by aakashthumaty** [wormhole-foundation/wormhole-connect#1428]
## Describe the current behavior in detail

<img width="602" alt="Screenshot 2024-01-05 at 11 38 39 AM" src="https://github.com/wormhole-foundation/wormhole-connect/assets/4761680/61eb50cf-c0cb-46bc-9332-26916c2d9565">

Today we show a modal as soon as the user enters the amount. 

## What is the expected/desired behavior

Instead of showing the full modal, when the user selects an amount we should show a yellow warning sign informing the user of the governor limit in place. The warning sign should follow the design model of the yellow warning box in the screenshot below but the box should be placed under the transfer amount "to" box, above the "routes" box. The larger modal, the one we currently pop up disruptively to the user we should show only once a user clicks approve and transfer. 

<img width="371" alt="Screenshot 2024-01-05 at 11 39 02 AM" src="https://github.com/wormhole-foundation/wormhole-connect/assets/4761680/8ff88ac1-bf8e-44fe-ad44-921e156ea65a">

The copy for this new yellow warning label:

"This transfer will take 24 hrs to complete as Wormhole has reached the daily transfer limit for [chain]. Learn more [link] about this temporary security measure."

**Additional changes:**
- [chain] should also be properly capitalized (it is lowercase right now)
- We should also put some thought into the interaction of Gateway and Gateway-connected chains. Gateway has a single aggregate governor limit that impacts Osmosis, Kujira, Evmos.

**Comment by tonyjin** [comment]
Added a comment to specify that the chain name should be properly capitalized. `base` -> `Base`. `xpla` -> `XPLA`

**Comment by tonyjin** [comment]
We should also put some thought into the interaction of Gateway and Gateway-connected chains. Gateway has a single aggregate governor limit that impacts Osmosis, Kujira, Evmos.

**Comment by aadam-10** [comment]
On hold until governor limits are hit somewhat less frequently

**Comment by aadam-10** [comment]
should be part of the redesign
